src/pages/index.js

```diff
@@ -1,13 +1,25 @@
 import React from "react"
 import SEO from "../components/SEO"
-import Hero from "../components/Hero/hero"
-import ArticlesSection from "../components/Articles/articles-section"
+import Hero from "../molecules/Hero"
+import FeaturedSection from "../atoms/FeaturedSection"
 
 const HomePage = () => (
   <>
     <SEO />
     <Hero />
-    <ArticlesSection />
+    <FeaturedSection level="1" title="Para quem é o Educodar?">
```

Comment:
`level` should be a number, so we could use:

```jsx
<FeaturedSection level={1} title="Para quem é o Educodar">
```

Reply:
Done.